spec/app/config/routes.rb

```diff
@@ -2,4 +2,6 @@
 
 Rails.application.routes.draw do
   root 'root#index'
+
+  resource :test, controller: :test, only: [:destroy]
```

Comment:
You shouldn't need to specify `controller` if it matches the name of the resource. For details, see: https://guides.rubyonrails.org/routing.html